---a/net/minecraft/entity/item/EntityMinecartChest.java
+++b/net/minecraft/entity/item/EntityMinecartChest.java
@@ -12,54 +12,45 @@
 import net.minecraft.util.EnumFacing;
 import net.minecraft.world.World;
 
-public class EntityMinecartChest extends EntityMinecartContainer
-{
+public class EntityMinecartChest extends EntityMinecartContainer {
+
     private static final String __OBFID = "CL_00001671";
 
-    public EntityMinecartChest(World worldIn)
-    {
+    public EntityMinecartChest(World worldIn) {
         super(worldIn);
     }
 
-    public EntityMinecartChest(World worldIn, double p_i1715_2_, double p_i1715_4_, double p_i1715_6_)
-    {
+    public EntityMinecartChest(World worldIn, double p_i1715_2_, double p_i1715_4_, double p_i1715_6_) {
         super(worldIn, p_i1715_2_, p_i1715_4_, p_i1715_6_);
     }
 
-    public void killMinecart(DamageSource p_94095_1_)
-    {
+    public void killMinecart(DamageSource p_94095_1_) {
         super.killMinecart(p_94095_1_);
         this.dropItemWithOffset(Item.getItemFromBlock(Blocks.chest), 1, 0.0F);
     }
 
-    public int getSizeInventory()
-    {
+    public int getSizeInventory() {
         return 27;
     }
 
-    public EntityMinecart.EnumMinecartType getMinecartType()
-    {
+    public EntityMinecart.EnumMinecartType getMinecartType() {
         return EntityMinecart.EnumMinecartType.CHEST;
     }
 
-    public IBlockState getDefaultDisplayTile()
-    {
+    public IBlockState getDefaultDisplayTile() {
         return Blocks.chest.getDefaultState().withProperty(BlockChest.FACING, EnumFacing.NORTH);
     }
 
-    public int getDefaultDisplayTileOffset()
-    {
+    public int getDefaultDisplayTileOffset() {
         return 8;
     }
 
-    public String getGuiID()
-    {
+    public String getGuiID() {
         return "minecraft:chest";
     }
 
-    public Container createContainer(InventoryPlayer playerInventory, EntityPlayer playerIn)
-    {
+    public Container createContainer(InventoryPlayer playerInventory, EntityPlayer playerIn) {
         return new ContainerChest(playerInventory, this, playerIn);
     }
-}
+}
